src/backend/distributed/sql/udfs/citus_isolation_test_session_is_blocked_skip_self_local_blocks/11.1-1.sql

```diff
@@ -0,0 +1,57 @@
+CREATE OR REPLACE FUNCTION pg_catalog.citus_isolation_test_session_is_blocked_skip_self_local_blocks(pBlockedPid integer, pInterestingPids integer[])
```

Comment:
these UDFs are not relevant for users. Is there a way to only define these for tests?

Reply:
I moved the definitions to `isolation_setup.spec`